src/main/resources/META-INF/optional/io.snyk.snyk-intellij-plugin-withJavaScript.xml

```diff
@@ -0,0 +1,5 @@
+<idea-plugin>
+  <extensions defaultExtensionNs="com.intellij">
+    <externalAnnotator language="JavaScript" implementationClass="snyk.code.annotator.SnykCodeAnnotator"/>
```

Comment:
Can we add TypeScript as well?

Reply:
Love the question - I THINK, it is covered by JavaScript. But this kind of information is next to impossible to find in the documentation. That said, let's try it out.

Reply:
`language="TypeScript"` for .ts files
`language="TypeScript  JSX"` for .tsx files
see [here ](https://github.com/DeepCodeAI/jetbrains-plugin/blob/e228308541c5d0bfb13156f219095859af33f683/src/main/resources/META-INF/plugin.xml#L33)what I did find experimentally ~one year ago